src/pages/History.svelte

```diff
@@ -26,63 +53,132 @@
 			if (x && x.funded) fundedAmount++;
 			if (x && x.claimed) claimedAmount++;
 		});
+		updateTitle();
+	});
 
-		document.title = `History - ${claimedAmount}/${fundedAmount} claimed/funded`;
-		if (cashlinks.length === fundedAmount && cashlinks.length === claimedAmount) {
-			stillUpdating = false;
-			try {
-				// Cashlinks in History
-				if (cashlinks.length && !isDev)
-					analyticsInstance.action("a883e4a7-3c86-4595-9d6d-31c5bccc48eb", { key: "Cashlinks", value: cashlinks.length });
-			} catch (e) {
-				// TODO: Add Sentry logs
+	const transactionsUnsubscribe = transactions.subscribe((txArray) => {
+		if (allFunded) return;
+		fundedAmount = 0;
+		txArray.forEach(async (txDetails: ClientTransactionDetails) => {
+			if (txDetails) {
```

Comment:
this is ok. But (imo) rather than creating a new code block like:
```js
if(boolean) {
    // code block
}
```
I rather prefer doing this:
```js
if(!boolean) return
// code block
```
So one tab less